python/qpid_dispatch/management/qdrouter.json

```diff
@@ -845,6 +845,45 @@
             }
         },
 
+        "logStats": {
+            "description": "histogram of the different severity-levels of events on the given log.",
+            "extends": "operationalEntity",
+            "attributes": {
+                "id": {
```

Comment:
Each entity has a built-in "name" and "identity" attribute.  You should use these instead of adding a new one.